frontend/src/views/Collection/style.ts

```diff
@@ -0,0 +1,67 @@
+import { Button } from "@blueprintjs/core";
+import styled from "styled-components";
+
+export const ViewGrid = styled.div`
```

Comment:
I got rid of the `margin-top` since it creates extra white space between the content and the header. I suppose it was there for the breadcrumbs?

Reply:
Resolve here! https://github.com/chanzuckerberg/corpora-data-portal/pull/689#discussion_r517602059